### PR TITLE
Fixed issue #20154 Multiple Store Views 2.3 not allowing unique descriptions per store view fields disabled

### PR DESCRIPTION
**Pull request by GovindaSharma**
### Description (*)
Fixed issue #20154 Multiple Store Views 2.3 not allowing unique descriptions per store view fields disabled

### Fixed Issues (if relevant)

1. magento/magento2 #20154: Multiple Store Views 2.3 not allowing unique descriptions per store view fields disabled

### Manual testing scenarios (*)

    1.Install Magento 2.3 via command line composer
    2.Run command line install
    3.Added 3 websites, 1 store and 1 store view each
    4.Add a sku to 1 store view. Title, price, description, url and save.
    5.Add that sku to another store view by checking the box in product editor.
    5.copy the data from 1 store view or default.
    6.try editing the sku from each store view.
    7.Description & short Description boxes are disabled even if you have use default is unchecked. but .

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3896](https://jira.corp.magento.com/browse/ENGCOM-3896) has been created to process this Pull Request

**Comment by sivaschenko** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, here is your [new Magento instance](https://pr-20357.instances.magento-community.engineering).
Admin access: [https://pr-20357.instances.magento-community.engineering/admin](https://pr-20357.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by sivaschenko** [comment]
Hi @GovindaSharma thanks for your contribution! Please check the test instance I've launched. The short description field is editable after going through the testing scenario, however, Description field is still disabled - I am unable to edit it.

**Comment by GovindaSharma** [comment]
@sivaschenko yes you are right,since i made changes on my local environment and it worked so accordingly i updated here,but unfortunately this is not working in server.So do you have any idea where should i look for file to resolve this issue? it seems we have to change in js file but not getting that file.So if you have any idea about can you please update me ? As this issue is very big so need to be resolved on instant basis.Any help from your side will be helpful.
thanks.

**Comment by GovindaSharma** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @GovindaSharma, here is your [new Magento instance](https://pr-20357.instances.magento-community.engineering).
Admin access: [https://pr-20357.instances.magento-community.engineering/admin](https://pr-20357.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by GovindaSharma** [comment]
Hi @sivaschenko ,i have fixed the issue now.You can check in test instance, description field is being editable now.

**Comment by GovindaSharma** [comment]
Hi @okorshenko just wanted to know why it is been removed from 2.3.1 milestone?

**Comment by GovindaSharma** [comment]
@sivaschenko any update on this?

**Comment by okorshenko** [comment]
Hi @GovindaSharma 
We will assign this PR to milestone once it is merged.

**Comment by GovindaSharma** [comment]
@okorshenko will it merge in 2.3.1 milestone?

**Comment by okorshenko** [comment]
Hi @GovindaSharma 
This PR has very high chances to be merged into 2.3.1 👍

**Comment by GovindaSharma** [comment]
@okorshenko thanks for your update , will wait for this pr to be merge soon.

**Comment by sivaschenko** [comment]
HI @GovindaSharma thanks for the update, on the latest code base I am able to edit the description even when "Use Default Value" is checked. The field should not be editable until the "Use Default Value" checkbox is unchecked

**Comment by sivaschenko** [comment]
Closing this pull request due to inactivity, feel free to reopen if you'd like to continue progress

**Comment by Axel29** [comment]
Hello,

I can confirm that the fix proposed in this PR works in my environment. However, since this JS doesn't use "Component", is there a way to override it until the 2.3.1 (or version that integrates this PR) is released?

Thank you.

**Pull request closed without merging**
